client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -684,72 +624,7 @@ export class CheckoutThankYou extends Component<
 					<PlanThankYouCard siteId={ this.props.selectedSite?.ID ?? 0 } { ...planProps } />
 				</Main>
 			);
-		} else if ( wasDomainProduct && ! wasBulkDomainTransfer ) {
```

Comment:
Domain only purchases are handled here: https://github.com/Automattic/wp-calypso/blob/05caf22fe6e33b861c937eaabf113616d9fe7896/client/my-sites/checkout/checkout-thank-you/index.tsx#L604-L605